data/mods/aftershock_exoplanet/setting_blacklists/location_blacklist.json

```
@@ -0,0 +1,89 @@
+[
+  {
+    "//": "mostly removes areas with their own storylines, static NPCs, and the like. some will be readded with time",
+    "type": "overmap_special",
+    "id": "Necropolis",
+    "overmaps": [  ],
+    "occurrences": [ 0, 0 ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "St_Johns_farm",
+    "overmaps": [  ],
+    "occurrences": [ 0, 0 ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Strangle Temple",
```

```suggestion
    "id": "Strange Temple",
```
It might be Strangle I can't remember but I thought it was strange

---

Under specials.json it is in fact strangle temple, yes